zkevm-circuits/src/evm_circuit/witness.rs

```diff
@@ -283,6 +283,28 @@ pub enum StepAuxiliaryData {
     },
 }
 
+impl From<circuit_input_builder::StepAuxiliaryData> for StepAuxiliaryData {
```

Comment:
We can probably directly use `StepAuxiliaryData` from the `circuit_input_builder` to avoid such conversion